src/utils/fileEncoding.ts

```diff
@@ -19,14 +19,14 @@ export function convertToBase64String(file: File): Promise<string> {
   });
 }
 
-interface FileField {
+export interface FileField {
   file: File;
```

Comment:
Is there a need for this interface, if there's only one field?

Reply:
It does just because it describes the structure of the AntD file field and its used in multiple places